### PR TITLE
1426 updating introduction code examples

### PR DESCRIPTION
**Pull request by stuartjnelson**
This is my first PR for the project so please let me know if I've not followed the workflow correctly!

### CODE REVIEW

**Comment by scottaohara** [comment]
Note: Fixes #1426

File: sections/introduction.include [hunk 1]
```diff
@@ -627,23 +633,23 @@
...
+      <style>
+        body {
+          background: blue;
+          color: yellow;
+        }
+      </style>
+    </head>
+    <body>
+      <h1>Styled page</h1>
+      <p>This is a paragraph has yellow text and the body element has a blue background.</p>
```
Comment (scottaohara) [inline sections/introduction.include]: Appears the paragraph is missing a word 
>"This is a paragraph **_which_** has yellow..." 

But if we're going to change the text to explicitly call out the visual styling, it should probably say something like:
>"This document has yellow text and a blue background"

It's not just the `p`, but the `h1` which would have the yellow text.

File: sections/introduction.include [hunk 2]
```diff
@@ -456,17 +456,18 @@
...
-      <body>
-        <h1>Sample page</h1>
-        <p>This is a <a href="demo.html">simple</a> sample.</p>
-        <!-- this is a comment -->
-      </body>
-    </html>
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <title>Document title</title>
```
Comment (scottaohara) [inline sections/introduction.include:462]: There are other sections of the prose where the original title "sample page" was referenced.  These would need to be updated.

File: sections/introduction.include (hunk 1 above)
Comment (stuartjnelson) [reply sections/introduction.include]: Thanks for the feedback @scottaohara. I've just pushed an update for both of these comments.

**Comment by scottaohara** [comment]
Marked as non substantive for IPR from ash-nazg.

File: sections/introduction.include [hunk 3]
```diff
@@ -627,23 +633,23 @@
...
+      <style>
+        body {
+          background: blue;
+          color: yellow;
+        }
+      </style>
+    </head>
+    <body>
+      <h1>Styled page</h1>
+      <p>This is document has yellow text and a blue background.</p>
```
Comment (scottaohara) [inline sections/introduction.include]: "This is document" should read "This document"

Comment (stuartjnelson) [reply sections/introduction.include]: Hi @scottaohara, I've just fixed that typo and pushed an update

**Review by chaals — COMMENTED** [review]
LGTM as a second review, so when @scottaohara is happy it seems good to go. Thank you for the contribution!

File: sections/introduction.include [hunk 4]
```diff
@@ -539,7 +539,7 @@
             <li class="t1">
               <{title}>
               <ul>
-                <li class="t3"><code>#text</code>: <span>Sample page</span></li>
```
Comment (isaacrojas0394) [inline sections/introduction.include:536]: Titulo del documento

**Pull request merged**